packages/experimental/src/entity/Entity.ts

```diff
@@ -358,3 +377,14 @@ if (process.env.NODE_ENV !== 'production') {
 export function isEntity(schema: Schema): schema is typeof Entity {
   return schema !== null && (schema as any).pk !== undefined;
 }
+
+function indexFromParams<I extends string>(
+  params: Readonly<object>,
+  indexes?: Readonly<I[]>,
+) {
+  if (!indexes) return undefined;
+  for (const index of indexes) {
```

Comment:
it’d transpile a lot smaller to use `return indexes.find((index) => index in params)` (but also, shouldn’t this use hasOwnProperty?)